addons/auth/README.md

```diff
@@ -6,17 +6,11 @@ This addon contains the authentication layer for Boundary.
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [Add to an App](#add-to-an-app)
-- [Installation](#installation)
-- [Yarn Commands](#yarn-commands)
-- [Linting](#linting)
-- [Formatting](#formatting)
-- [Running tests](#running-tests)
-- [Running the dummy application](#running-the-dummy-application)
-- [Contributing](#contributing)
-  - [Building ToC](#building-toc)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Compatibility
```

Comment:
This is one of the changes the script does and actually breaks our Readme.

Can you revert this to the table of contents? Perhaps just running `yarn run doc:toc` makes it.